### ISSUE TITLE
Tips weirdness in Spaceland, LA

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#19]
```
1.) Had a force close happen: brought up "Spaceland" in LA which has
two tips. I had the keyboard open so it was in horizontal. One tip
didn't show all the way so I touch-held to try to drag it but both
tips immediately disappeared and "tips" was replaced by an offset
"todos" bar. Seconds after this, the app force closes.
```

Original issue reported on code.google.com by `jlapenna` on 8 Aug 2009 at 10:53

**Comment by GoogleCodeExporter** [comment]
```
Want to fix this report a little. The bar never says "tips", it always says 
Todos.

I can encounter this bug regardless of horizontal or vertical orientation.

Steps to reproduce:

1.) Search "Spaceland".
2.) Click on "Spaceland".
3.) Touch-drag on either "Todo".

Both Todos will disappear, leaving the GUI line elements, and a few seconds 
later the
application will force close.
```

Original comment by `amiantos@gmail.com` on 8 Aug 2009 at 4:02

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 865e2e4154.
```

Original comment by `jlapenna` on 10 Aug 2009 at 2:33
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 865e2e4154.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:46
